File: 디자인패턴/java 디자인 패턴/Bridge/SimpleDisplay.java
public class SimpleDisplay implements Display {
    public SimpleDisplay() {
        System.out.println("[Style - SimpleDisplay]");
    }

    @Override
    public void title(Draft draft) {
        System.out.println(draft.getTitle());
    }
    @Override
    public void author(Draft draft) {
        System.out.println(draft.getAuthor());
    }
    @Override
    public void content(Draft draft) {
        for (String item : draft.getContent()) {
            System.out.println(item);
        }
    }
}

File: 디자인패턴/Python 디자인 패턴/Bridge/SimpleDisplay.py
from Display import*
from Draft import*
class SimpleDisplay(Display):
    def __init__(self) -> None:
        print("[Style - SimpleDisplay]")
        
    def title(this, draft:Draft):
        print(draft.getTitle())

    def author(this, draft:Draft):
        print(draft.getAuthor())

    def content(this, draft:Draft):
        for item in draft.getContent():
            print(item)
    pass